gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/v4/impl/ApiEntrypointServiceImpl.java

```diff
@@ -155,6 +157,15 @@ private List<ApiEntrypointEntity> getEntrypoints(
                         .stream()
                 )
                 .toList();
+        } else if (genericApiEntity.getDefinitionVersion() == DefinitionVersion.V4 && genericApiEntity instanceof NativeApiEntity api) {
+            return api
+                .getListeners()
+                .stream()
+                .map(listener -> {
+                    var kafkaListener = (KafkaListener) listener;
+                    return getKafkaNativeApiEntrypointEntity(kafkaListener.getHost(), kafkaListener.getPort(), tagEntrypoints);
```

Comment:
```suggestion
// FIXME: use the kafkaPort config from properties when available (like tcpPort)
                    return getKafkaNativeApiEntrypointEntity(kafkaListener.getHost(), 9092), tagEntrypoints);
```

the getPort here must not be used. it's for another concept, but not necessary for 4.6.

Reply:
Done